packages/notification-center/rollup.config.mjs

```diff
@@ -17,6 +17,7 @@ export default [
         file: packageJson.main,
         format: 'cjs',
         sourcemap: true,
+        interop: 'auto',
```

Comment:
The main issue was with the `@emotion/styled` which exports the `styled.div` functions in the default export, which works in the ESM module, but not in CJS which natively doesn't support it well.

The option from the docs:
```
Controls how Rollup handles default, namespace and dynamic imports from external dependencies in formats like CommonJS that do not natively support these concepts.

"auto" combines both "esModule" and "default" by injecting helpers that contain code that detects at runtime if the required value contains the [__esModule property](https://rollupjs.org/guide/en/#outputesmodule).
```